packages/eas-cli/src/graphql/queries/BranchQuery.ts

```diff
@@ -0,0 +1,33 @@
+import gql from 'graphql-tag';
+
+import { graphqlClient, withErrorHandlingAsync } from '../client';
+import { ViewBranchQuery, ViewBranchQueryVariables } from '../generated';
+
+export const BranchQuery = {
+  async getBranchByNameAsync({ appId, name }: { appId: string; name: string }) {
+    return await withErrorHandlingAsync<ViewBranchQuery>(
```

Comment:
Maybe this function should return the app object without the surrounding object.

I mean:
```
{
  id: ...,
  updateBranchByName: {
  }
}
```
instead of:
```
{
  app: {
    id: ...,
    updateBranchByName: {
    }
  }
}
```

Reply:
It'd look a lot cleaner this way. Especially because `app` is actually the only thing the query returns.

Reply:
💯